jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/exceptions/exceptionhandling/ApiResponseExceptionHandler.java

```diff
@@ -42,9 +45,8 @@ public ResponseEntity<ApiErrorResponse> handleTypeMismatchException(TypeMismatch
 
     @ExceptionHandler(PersistenceException.class)
     public ResponseEntity<ApiErrorResponse> handlePersistenceException(PersistenceException ex) {
-        LOGGER.severe("Persistence exception occurred whilst saving a resource. " + ex.getMessage());
-        var msg = "There was an error persisting data.";
-        return createResponseEntity(msg, HttpStatus.BAD_REQUEST);
+        LOGGER.severe(SERVER_ERROR + ex.getMessage());
```

Comment:
extremely minor, not necessary to update: there is no space between the two sentences/messages here when they're concatenated.
I'll resolve and approve anyway, just mentioning.